### ISSUE TITLE
Change ExcessIceStreams test mod to ExcessIceStreamsCold to also do a cold start?

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/CTSM#2672]
When excess ice is turned on by default for clm6_0 the main use of excess ice streams will be for cold-start cases. Hence, it makes sense to me that the ExcessceStreams test mod should be changed to also do a cold-start. Since, it'll appear as a new test, this could be done on the b4b-dev branch.

**Comment by ekluzek** [comment]
In terms of evaluating the excess ice tests. Right now we only have two excess ice tests in the aux_clm test list and one for ctsm_sci. However, when excess ice is turned on for clm6_0, more tests will be exercising excess ice and so the three tests that are specific for excess ice might be sufficient, with all of the clm6_0 tests also exercising excess ice.

**Comment by ekluzek** [comment]
@mvdebolskiy thoughts on this?

**Comment by mvdebolskiy** [comment]
It will get tested automatically when it is default on any ColdStart tests. I would rather add 2 tests where it is turned off on the cold start (for temperature testing) and one with finidat with no excess ice vars and streams turned on (since we want to keep this possibility).

**Comment by ekluzek** [comment]
Good point @mvdebolskiy I changed the title and added to the text at the top to lay out a plan for doing what you suggest. Feel free to add more feedback...
